packages/core/useMediaQuery/index.ts

```diff
@@ -58,5 +58,5 @@ export function useMediaQuery(query: MaybeRefOrGetter<string>, options: Configur
     mediaQuery = undefined
   })
 
-  return matches
+  return computed(() => matches.value)
```

Comment:
could it make more sense to wrap it in a readonly instead?

```suggestion
return readonly(matches)
```

presumably a `computed` is so you can compute something, whereas `readonly` will just wrap an existing ref as-is

(i could be wrong, tell me if i am!)

Reply:
according to the vue docs, 

>  Takes an object (reactive or plain) or a ref and returns a readonly proxy to the original.

`readonly` is more proper for a `reactive` object.

and `computed` is lazy...

Reply:
Yeah, I think both should work in this case. In terms of memory usage, `readonly` is definitely a bit more efficient. But the benefit of `computed` is that it deduces the value and guards unchanged effects to be triggered.

Reply:
But in this case the effects have already triggered because it's a wrapped ref, there is no deducing or effects to track, so readonly does make more sense